src/vivarium_conic_sqlns/components/sq_lns_intervention.py

```diff
@@ -111,6 +106,7 @@ def name(self):
         return f'sqlns_effect_on_{self.target.name}'
 
     def setup(self, builder):
+        self.treatment_duration = builder.configuration.sqlns.duration
```

Comment:
do you need treatment_duration? it seems like you use treatment_start/end instead?